Dockerfile

```
@@ -36,6 +34,24 @@ COPY --from=voila_rootfs_builder /etc/ /etc/
 
 
 FROM base_image_python AS kernel_rootfs_builder
```

I think this look this now, because python installed with apt is no longer needed.

```suggestion
FROM base_image AS kernel_rootfs_builder
```